package/network/utils/iproute2/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=iproute2
-PKG_VERSION:=6.0.0
+PKG_VERSION:=v6.1.0
```

Comment:
Will this break OPKG?

Maybe keep the version without v and just add the v in `PKG_SOURCE`

Reply:
Changed that